src/main/java/dev/slimevr/vr/processor/skeleton/LegTweakBuffer.java

```diff
@@ -12,8 +12,9 @@
  * rules: The conditions for an unlock are as follows: 1. the foot is to far
  * from its correct position 2. a velocity higher than a threashold is achived
  * 3. a large acceleration is applied to the foot 4. angular velocity of the
- * foot goes higher than a threashold The conditions for a lock are the opposite
- * of the above but require a lower value for all of the above conditions
+ * foot goes higher than a threashold. The conditions for a lock are the
```

Comment:
```suggestion
 * foot goes higher than a threshold. The conditions for a lock are the
```
Smol misspelled word.